### PR TITLE
Marcio/registrar content strategy update

### PR DESCRIPTION
**Pull request by marciocloudflare**
Updated pages according to PCX's content strategy

### CODE REVIEW

File: products/registrar/src/content/account-options/enabling-dnssec.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 title: Enabling DNSSEC
-order:
```
Comment (pedrosousa) [inline products/registrar/src/content/account-options/enabling-dnssec.md:3]: Can we use active verbs in headings? "Enable DNSSEC", "Confirming DNSSEC".

File: products/registrar/src/content/account-options/transfer-out-from-cloudflare.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 order:
-pcx-content-type: tutorial
+pcx-content-type: how-to
 ---
 
 # Transfer out from cloudflare
```
Comment (pedrosousa) [inline products/registrar/src/content/account-options/transfer-out-from-cloudflare.md]:
```suggestion
# Transfer out from Cloudflare
```
I believe the heading is not entirely clear... Maybe "Transfer your domain to another registrar"?

File: products/registrar/src/content/faq.md [hunk 3]
```diff
@@ -6,31 +6,35 @@ pcx-content-type: faq
 
 # Frequently Asked Questions
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
# FAQ
``` 
According to our content strategy, the title should be "FAQ". The same for the sidebar title.

File: products/registrar/src/content/faq.md [hunk 4]
```diff
@@ -6,31 +6,35 @@ pcx-content-type: faq
...
 
-## After transferring my domain to Cloudflare, my domain’s expiration date changed. Why?
+Below you will find answers to our most commonly asked questions on Cloudflare Registrar, how billing works, transfers and how to troubleshoot common issues you may come across. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
Below you will find answers to our most commonly asked questions on Cloudflare Registrar, namely about billing, domain transfers, and how to troubleshoot common issues you may come across. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.
```

File: products/registrar/src/content/faq.md [hunk 5]
```diff
@@ -6,31 +6,35 @@ pcx-content-type: faq
...
 
-ICANN requires that any transfer also extends the expiration date of your domain by at least one year — that is one year from your current expiration date, not one year from the date of transfer. For example, if you transfer a domain on October 10, 2020, but it expires on March 10, 2021, your new expiration date will be March 10, 2022.
+* [Domain transfers](#why-did-my-transfer-fail)
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]: Any reason for highlighting these three specific entries? They're not categories.

File: products/registrar/src/content/faq.md [hunk 6]
```diff
@@ -6,31 +6,35 @@ pcx-content-type: faq
...
 
-Whenever a domain is first registered, the registrant purchases control of that domain for some number of years — up to 10 years. For example, a domain registered on October 8, 2020 will have an expiration date of October 8th in some year between 2021 and 2030, depending on the amount of years originally purchased.
+## Why did my transfer fail?
 
-Transferring a domain adds time to the current expiration date. Time purchased is never lost. When you transfer your domain to Cloudflare, you are purchasing an additional year of registration on top of your current expiration.
+Domain transfers sometimes fail. Your account team can help you address what failed so that you can complete the transfer to Cloudflare.
 
-## When will I be billed?
+1. Make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from your **Account Home** > **Registrar** > **Transfer** page.
+1. You can reattempt the transfer from the transfer launch page.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]: Maybe "retry" instead of "reattempt" (unless it's what appears in the UI)?

File: products/registrar/src/content/faq.md [hunk 7]
```diff
@@ -6,31 +6,35 @@ pcx-content-type: faq
...
+If you cannot solve the issue, open a support ticket or contact your account team.
 
-## My domain’s registration wasn’t extended by one year after transferring to Cloudflare
 
-When you transfer your domain to Cloudflare, the registry will extend your registration by one year. However, in one specific circumstance your transfer could result in you keeping your original expiration date.
+## Why did my domain's expiration date changed after transferring it to Cloudflare?
 
-When a domain expires, the registration enters the auto-renew grace period. During that time, you can renew the domain at your registrar to avoid losing it. If your domain expires at your current registrar, you renew it and then transfer to Cloudflare within 45 days, the registry can restrict the addition of an extra year.
+ICANN requires that any transfer also extends the expiration date of your domain by at least one year — that is one year from your current expiration date, not one year from the date of transfer. For example, if you transfer a domain on October 10, 2020, but it expires on March 10, 2021, your new expiration date will be March 10, 2022.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
ICANN requires that any transfer also extends the expiration date of your domain by at least one year — that is one year from your current expiration date, not one year from the date of transfer. For example, if you transfer a domain on October 10, 2021, but it expires on March 10, 2022, your new expiration date will be March 10, 2023.
```
Expiration should be in the future, so I suggest that we update the dates to 2021-2023.

File: products/registrar/src/content/faq.md [hunk 8]
```diff
@@ -40,34 +44,41 @@ To see the progress of your transfer, log into the Cloudflare dashboard and sele
...
-
-If you cannot resolve the issue, open a support ticket or contact your account team.
 
 ## Why am I not allowed to transfer my domain?
 
 ICANN prohibits domain transfers within 60 days of a change to the WHOIS data or registrar of a domain. If you modified your contact information, transferred registrars, or registered your domain in the last 60 days, Cloudflare will be unable to process your transfer immediately.
 
 You can leave the domain **In Progress** and Cloudflare will attempt to process the transfer until the 60-day window passes.
 
+
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]: We don't need this empty line.

File: products/registrar/src/content/faq.md [hunk 9]
```diff
@@ -40,34 +44,41 @@ To see the progress of your transfer, log into the Cloudflare dashboard and sele
...
 ## Why am I not able to start a transfer?
 
 If you have an <a href='https://support.cloudflare.com/hc/articles/203471284#h_1l0KGygoBX9QYjNrhAcHjg'>unverified email address</a>, you might experience issues when initiating a domain transfer.
 
-## Premium domains
+
+## What happens if I entered the wrong auth code?
+
+If you input the incorrect auth code (also referred to as authentication code and authorization code), return to the **Domain Registration** page or the **Overview** for your site. You can use the available input field to reenter your authentication code.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
If you enter an incorrect auth code (also referred to as authentication code or authorization code), return to the **Domain Registration** page or the **Overview** for your site. You can use the available input field to reenter your authentication code.
```

File: products/registrar/src/content/faq.md [hunk 10]
```diff
@@ -40,34 +44,41 @@ To see the progress of your transfer, log into the Cloudflare dashboard and sele
...
 You can leave the domain **In Progress** and Cloudflare will attempt to process the transfer until the 60-day window passes.
 
+
 ## Why am I not able to start a transfer?
 
 If you have an <a href='https://support.cloudflare.com/hc/articles/203471284#h_1l0KGygoBX9QYjNrhAcHjg'>unverified email address</a>, you might experience issues when initiating a domain transfer.
 
-## Premium domains
+
+## What happens if I entered the wrong auth code?
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
## What happens if I enter the wrong auth code?
```
Alternatively, "What should I do if I entered an incorrect auth code?"

File: products/registrar/src/content/faq.md [hunk 11]
```diff
@@ -40,34 +44,41 @@ To see the progress of your transfer, log into the Cloudflare dashboard and sele
...
 
+
+## My domain’s registration was not extended by one year after transferring to Cloudflare
+
+When you transfer your domain to Cloudflare, the registry will extend your registration by one year. However, in one specific circumstance your transfer could result in you keeping your original expiration date.
+
+When a domain expires, the registration enters the auto-renew grace period. During that time, you can renew the domain at your registrar to avoid losing it. If your domain expires at your current registrar, you renew it and then transfer to Cloudflare within 45 days, the registry can restrict the addition of an extra year.
+
+Say you have `example.com` registered and it expires on December 10, 2020. You decide to renew it during the auto-renew grace period on December 20, 2020. That renewal extends the registration to December 20, 2021. You then transfer to Cloudflare on December 30, 2020. Since that transfer is within 45 days of the expiration, the registry may not add the year to your registration. When you transfer to Cloudflare or any registrar in this circumstance, your expiration can still remain December 20th, 2021.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
Say you have `example.com` registered and it expires on December 10, 2021. You decide to renew it during the auto-renew grace period on December 20, 2021. That renewal extends the registration to December 20, 2022. You then transfer to Cloudflare on December 30, 2021. Since that transfer is within 45 days of the expiration, the registry may not add the year to your registration. When you transfer to Cloudflare or any registrar in this circumstance, your expiration can still remain December 20th, 2022.
```

File: products/registrar/src/content/faq.md [hunk 12]
```diff
@@ -40,34 +44,41 @@ To see the progress of your transfer, log into the Cloudflare dashboard and sele
...
+
+
 ## What Happens When a Domain Expires?
 
 Cloudflare currently offers a 40 day grace period for most top-level domains (TLDs). During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Contact your account team for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
Cloudflare currently offers a 40-day grace period for most top-level domains (TLDs). During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Contact your account team for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.
```
(Just added a hyphen in `40-day`.)

File: products/registrar/src/content/faq.md [hunk 13]
```diff
@@ -78,4 +89,10 @@ In summary, here is what will happen after a domain expires:
...
 * **Day 41 - 70:** Redemption Period.
-* **Day 71 - 75:** Pending Delete Period.
\ No newline at end of file
+* **Day 71 - 75:** Pending Delete Period.
+
+---
+
+## When will I be billed?
+
+You will be billed when you input your authorization code and initiate the transfer of your domain to Cloudflare. Make sure to copy / paste the code to avoid mistakes. The transfer will not initiate if the code is incorrect.
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]:
```suggestion
You will be billed when you input your authorization code and initiate the transfer of your domain to Cloudflare. Make sure to copy and paste the code to avoid mistakes. The transfer will not initiate if the code is incorrect.
```

File: products/registrar/src/content/get-started/index.md [hunk 14]
```diff
@@ -0,0 +1,10 @@
+---
+order: 2
+pcx-content-type: navigation
+---
+
+# Get started
+
+To transfer your domain to Cloudflare, complete the following tasks.
```
Comment (pedrosousa) [inline products/registrar/src/content/get-started/index.md]:
```suggestion
To transfer your domain to Cloudflare, complete the following tasks:
```

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md [hunk 15]
```diff
@@ -9,6 +9,10 @@ import EmailForward from "../_partials/_email-forward.md"
 
 # How to transfer your domain to Cloudflare
 
+Transferring your domain to Cloudflare tell your registry that a different registrar can now set those authoritative records for you. The relationship is based on trust. Registries only trust one registrar at any given time to make changes on your behalf.
```
Comment (pedrosousa) [inline products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md]:
```suggestion
Transferring your domain to Cloudflare tells your registry that a different registrar can now set those authoritative records for you. The relationship is based on trust. Registries only trust one registrar at any given time to make changes on your behalf.
```

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md [hunk 16]
```diff
@@ -9,6 +9,10 @@ import EmailForward from "../_partials/_email-forward.md"
 
 # How to transfer your domain to Cloudflare
```
Comment (pedrosousa) [inline products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md]: Consider removing "How to" from the title.
Also, you can delete the empty `order:`.

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md [hunk 17]
```diff
@@ -9,6 +9,10 @@ import EmailForward from "../_partials/_email-forward.md"
...
 
+Transferring your domain to Cloudflare tell your registry that a different registrar can now set those authoritative records for you. The relationship is based on trust. Registries only trust one registrar at any given time to make changes on your behalf.
+
+Transferring a domain to a new registrar informs the registry that they should instead trust that new registrar to modify information. The process requires some action steps at your new and previous registrar. Each registrar handles transfers a bit differently, but in general, they follow a pattern based on rules set by ICANN, the organization responsible for regulating domain registration.
+
 This section contains generic details on how to transfer your domain to Cloudflare from most registrars.
```
Comment (pedrosousa) [inline products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md]:
```suggestion
This section contains generic instructions on how to transfer your domain to Cloudflare from most registrars.
```
Feels odd to say "generic details".

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md [hunk 18]
```diff
@@ -9,6 +9,10 @@ import EmailForward from "../_partials/_email-forward.md"
...
 
 <BeforeYouBegin/>
```
Comment (pedrosousa) [inline products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md:17]: (Cannot comment on the correct line.)
Line 37 - "1. **Log in** to"

File: products/registrar/src/content/index.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Overview
 order: 0
-pcx-content-type: landing-page
+pcx-content-type: overview
 ---
 
 # Cloudflare Registrar
```
Comment (pedrosousa) [inline products/registrar/src/content/index.md:7]: The "Setup domain transfer" button in this page seems to point to `products/registrar/src/content/setup-domain-transfers/index.md`, which was deleted in this PR.

File: products/registrar/src/content/why-choose-cloudflare/top-level-domains-supported.md [hunk 20]
```diff
@@ -5,6 +5,8 @@ pcx-content-type: reference
 
 # Top Level Domains supported
 
+In the [DNS](https://www.cloudflare.com/learning/ddos/glossary/domain-name-system-dns/) hierarchy, a top-level domain (TLD) represents the first stop after the root zone. In simpler terms, a TLD is everything that follows the final dot of a [domain name](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name/). For example, in the domain name ‘google.com’, ‘.com’ is the TLD. Some other popular TLDs include ‘.org’, ‘.uk’, and ‘.edu’.
```
Comment (pedrosousa) [inline products/registrar/src/content/why-choose-cloudflare/top-level-domains-supported.md]:
```suggestion
In the [DNS](https://www.cloudflare.com/learning/ddos/glossary/domain-name-system-dns/) hierarchy, a top-level domain (TLD) represents the first stop after the root zone. In simpler terms, a TLD is everything that follows the final dot of a [domain name](https://www.cloudflare.com/learning/dns/glossary/what-is-a-domain-name/). For example, in the domain name `google.com`, `.com` is the TLD. Some other popular TLDs include `.org`, `.uk`, and `.edu`.
```
Avoids getting an automatic link to Google, and makes all quoted text similar.

File: products/registrar/src/content/faq.md (hunk 5 above)
Comment (marciocloudflare) [reply products/registrar/src/content/faq.md]: Just because in our content strategy it states that for medium FAQs (more than 10) we should structure it into sections to facilitate readability and discoverability of content.

File: products/registrar/src/content/account-options/enabling-dnssec.md (hunk 1 above)
Comment (marciocloudflare) [reply products/registrar/src/content/account-options/enabling-dnssec.md:3]: done

File: products/registrar/src/content/faq.md (hunk 6 above)
Comment (marciocloudflare) [reply products/registrar/src/content/faq.md]: done

File: products/registrar/src/content/faq.md (hunk 8 above)
Comment (marciocloudflare) [reply products/registrar/src/content/faq.md]: done

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md (hunk 16 above)
Comment (marciocloudflare) [reply products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md]: done

File: products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md (hunk 18 above)
Comment (marciocloudflare) [reply products/registrar/src/content/get-started/transfer-domain-to-cloudflare.md:17]: done

File: products/registrar/src/content/index.md (hunk 19 above)
Comment (marciocloudflare) [reply products/registrar/src/content/index.md:7]: Good catch! Thanks

File: products/registrar/src/content/faq.md (hunk 5 above)
Comment (pedrosousa) [reply products/registrar/src/content/faq.md]: When using sections, I think it should link to a section and not to a specific question. In this case, section names would be level-2 headings, and questions would become level-3 headings.

Comment (marciocloudflare) [reply products/registrar/src/content/faq.md]: Should be addressed

File: products/registrar/src/content/faq.md [hunk 21]
```diff
@@ -4,84 +4,80 @@ order: 4
...
 Domain transfers sometimes fail. Your account team can help you address what failed so that you can complete the transfer to Cloudflare.
 
 1. Make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from your **Account Home** > **Registrar** > **Transfer** page.
-1. You can reattempt the transfer from the transfer launch page.
+1. You can retry the transfer from the transfer launch page.
 
 If you cannot solve the issue, open a support ticket or contact your account team.
 
-
 ## Why did my domain's expiration date changed after transferring it to Cloudflare?
```
Comment (pedrosousa) [inline products/registrar/src/content/faq.md]: Should this be a level-3 heading also, to be included in the "Domain transfers" section?

**Review by pedrosousa — APPROVED** [review]
